src/middleware/navigation/check.condition.ts

```diff
@@ -21,6 +21,17 @@ const checkHasAppData = (appData: ApplicationData): boolean => {
   return appData && Object.keys(appData).length !== 0;
 };
 
+const checkBOsDetailsEntered = (appData: ApplicationData): boolean => {
```

Comment:
I meant these are already defined elsewhere.

Reply:
in application.data.ts

export const checkBOsDetailsEntered = (appData: ApplicationData): boolean => {
  return Boolean( appData[BeneficialOwnerIndividualKey]?.length || appData[BeneficialOwnerOtherKey]?.length || appData[BeneficialOwnerGovKey]?.length );
};

export const checkMOsDetailsEntered = (appData: ApplicationData): boolean => {
  return Boolean( appData[ManagingOfficerKey]?.length || appData[ManagingOfficerCorporateKey]?.length ) ;
};

Reply:
Should have made that clear, sorry